src/Connectors/src/Connector/CosmosDb/CosmosDbTypeLocator.cs

```diff
@@ -3,8 +3,9 @@
 // See the LICENSE file in the project root for more information.
 
 using Steeltoe.Common.Reflection;
+using Steeltoe.Connector;
 
-namespace Steeltoe.Connector.CosmosDb;
+namespace Steeltoe.Bootstrap.AutoConfiguration.TypeLocators;
```

Comment:
My initial reaction to this change is that I don't agree with it. Bootstrap.Autoconfiguration is a completely optional layer on top of Connectors, and these classes are a core piece of Connectors. I could see centralizing all Assembly/Type metadata somewhere in Common maybe, but I'm not sure if that's worth the effort since these aren't likely to be used anywhere else. Outside the scope of this PR, but we might even be better served by marking these `internal`

Reply:
I forgot that we need to use these in AutoConfiguration... so that could explain why this change might make sense from a certain point of view, but I'm not sure I agree with that though

Reply:
I didn't realize these files were linked into the project from somewhere else. I've reverted the namespace changes. The types aren't being used from the Bootstrap.Autoconfiguration project, so I'm going to try to remove the linking and see if things still work.

Reply:
Tried to remove the linking and tested in an empty API project with management endpoints, using the NuGets produced from this PR. It throws a `FileNotFoundException` at startup, referring to the Steeltoe.Connector assembly. Then reverted the changes in this PR and repeated the test, which succeeded.

Why this matters is still a mystery to me, but somehow the linking appears to be essential, despite all unit tests succeeding without them.